src/decoder/plugins/SidplayDecoderPlugin.cxx

```diff
@@ -40,6 +40,7 @@
 #include <sidplayfp/builders/resid.h>
 #include <sidplayfp/builders/residfp.h>
 #include <sidplayfp/SidDatabase.h>
+#include <fs/io/FileReader.cxx>
```

Comment:
This is an internal header; should have double quotes instead of pointed quotes.